en/sim_gazebo_gz/README.md

```diff
@@ -31,16 +31,42 @@ sudo apt-get install gz-garden
 
 ## Running the Simulation
 
-Gazebo SITL simulation can be conveniently run through a `make` command as shown below:
+Gazebo SITL simulation can be conveniently run using a `make` command as shown below:
 
 ```sh
 cd /path/to/PX4-Autopilot
 make px4_sitl gz_x500
 ```
 
-This will run both the PX4 SITL instance and the Gazebo client.
+This runs both the PX4 SITL instance and the Gazebo client.
 Note that all gazebo make targets have the prefix `gz_`.
 
+Another way that Gazebo SITL can be connected is in _standalone mode_.
+In standalone mode PX4 SITL and Gazebo are started separately in their own terminals.
+You start PX4 in standalone mode by prefixing the make command with `GZ_PX4_STANDALONE=1`:
+
+```sh
+cd /path/to/PX4-Autopilot
+GZ_PX4_STANDALONE=1 make px4_sitl gz_x500
+```
+
+If you have not yet started _gz-server_ when you run the `make` command, you will see the following warning until gazebo has been started and an instance of _gz-server_ is detected by PX4:
+
+```sh
+WARN [gz bridge] Service call timed out as Gazebo has not been detected
+```
+
+The simplest way to start the simulation is to run the `simulation-gazebo` script found in the [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository.
+This will launch a _gz-server_ instance that can be started with any world and vehicle.
+For more information and arguments, see [Gazebo Models](./gazebo-models.md).
+
+The script can be started with:
+
+```sh
+cd /path/to/PX4-gazebo-models
+python3 simulation-gazebo
+```
```

Comment:
At no point have I been asked to download `PX4-gazebo-models` so this won't work. I am pretty sure this all gets copied into `/.simulation-gazebo` when you first run make with a gazebo target? Is this in the root of your current drive like it looks?

Reply:
I make a refernce to it here : `The simplest way to start the simulation is to run the `simulation-gazebo` script found in the [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository.` 

The thing is that you don't need to download the whole repository. You only really need this one file. Per default it is installed in root at `/.simulation-gazebo`. But you can of course provide a different path. But then you always need to set that path when you run the script.